src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/get_interior_vehicle_data_request.cc

```diff
@@ -147,6 +158,12 @@ void GetInteriorVehicleDataRequest::Execute() {
     return;
   }
 
+  if (!ProcessRateLimits()) {
```

Comment:
@ZhdanovP if this function does not send response in the body please rename it to `CheckRateLimits`